client/app/controllers/project.js

```diff
@@ -26,6 +26,11 @@ export default class ProjectController extends Controller {
     this.modalOpen = true;
   }
 
+  @action
+  removeEditor(applicant) {
+    applicant.destroyRecord();
+  }
```

Comment:
So are you actually going to delete the record or just deactivate it? I know that we don't show deactivated contact records in the project editor list based on new changes that are in develop, so maybe we could do the same thing for `projectApplicant`?

Reply:
@trbmcginnis deactivate it in CRM. However, to the frontend it is deleting it. 

Calling `destroyRecord()` will send a DELETE request to the backend, however the backend DELETE endpoint actually just updates the contact record to be inactive. 

Wait... isn't the project editor list the same as a list of project applicants?

Reply:
We call it a "project editor" in the frontend but the entity that it's referring to in the backend is `dcp_projectapplicants`

Reply:
@trbmcginnis is right here. It should not be a destroyRecord, it should be a regular patch request to update the field.